### ISSUE TITLE
Build error after updating to latest nightly

### ISSUE THREAD

**Issue by alexschneider** [rust-lang/log#26]
```
<log macros>:4:8: 4:21 error: mismatched types:
 expected `usize`,
    found `u32`
(expected usize,
    found u32) [E0308]
<log macros>:4 line : line ! (  ) , file : file ! (  ) , module_path : module_path ! (  ) , }
                      ^~~~~~~~~~~~~
<log macros>:4:8: 4:21 error: mismatched types:
 expected `usize`,
    found `u32`
(expected usize,
    found u32) [E0308]
<log macros>:4 line : line ! (  ) , file : file ! (  ) , module_path : module_path ! (  ) , }
                      ^~~~~~~~~~~~~
<log macros>:4:8: 4:21 error: mismatched types:
 expected `usize`,
    found `u32`
(expected usize,
    found u32) [E0308]
<log macros>:4 line : line ! (  ) , file : file ! (  ) , module_path : module_path ! (  ) , }
                      ^~~~~~~~~~~~~
<log macros>:4:8: 4:21 error: mismatched types:
 expected `usize`,
    found `u32`
(expected usize,
    found u32) [E0308]
<log macros>:4 line : line ! (  ) , file : file ! (  ) , module_path : module_path ! (  ) , }
                      ^~~~~~~~~~~~~
<log macros>:4:8: 4:21 error: mismatched types:
 expected `usize`,
    found `u32`
(expected usize,
    found u32) [E0308]
<log macros>:4 line : line ! (  ) , file : file ! (  ) , module_path : module_path ! (  ) , }
                      ^~~~~~~~~~~~~
<log macros>:4:8: 4:21 error: mismatched types:
 expected `usize`,
    found `u32`
(expected usize,
    found u32) [E0308]
<log macros>:4 line : line ! (  ) , file : file ! (  ) , module_path : module_path ! (  ) , }
                      ^~~~~~~~~~~~~
<log macros>:4:8: 4:21 error: mismatched types:
 expected `usize`,
    found `u32`
(expected usize,
    found u32) [E0308]
<log macros>:4 line : line ! (  ) , file : file ! (  ) , module_path : module_path ! (  ) , }
                      ^~~~~~~~~~~~~
<log macros>:4:8: 4:21 error: mismatched types:
 expected `usize`,
    found `u32`
(expected usize,
    found u32) [E0308]
<log macros>:4 line : line ! (  ) , file : file ! (  ) , module_path : module_path ! (  ) , }
                      ^~~~~~~~~~~~~
<log macros>:4:8: 4:21 error: mismatched types:
 expected `usize`,
    found `u32`
(expected usize,
    found u32) [E0308]
<log macros>:4 line : line ! (  ) , file : file ! (  ) , module_path : module_path ! (  ) , }
```

**Comment by alexcrichton** [comment]
Fixed in https://github.com/rust-lang/log/commit/e5b566ffb5cad2e28a255551cc838a92998dffc0, but thanks!

**Comment by AaronM04** [comment]
Is this fix in 0.2.5?
